### PR TITLE
EDU-1062 User profile page improvements

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1062

### CODE REVIEW

File: src/api-clients/user-api-client.js [hunk 1]
```diff
@@ -200,15 +200,6 @@ class UserApiClient {
...
 
-  getCreatedDocuments({ userId }) {
-    return this.httpClient
-      .get(
-        `/api/v1/users/${encodeURIComponent(userId)}/created-documents`,
-        { responseType: 'json' }
-      )
-      .then(res => res.data);
-  }
-
```
Comment (ahelmberger) [inline src/api-clients/user-api-client.js:211]: Nice that you put the replacement in to a better fitting place :+1:

**Comment by ahelmberger** [comment]
Also: nice consolidation of the small font-size!

File: src/components/pages/user-profile.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+accountClosed:
+  en: This user account was closed.
+  de: Dieses Benutzerkonto wurde geschlossen.
+documentsHeadline:
+  en: Recent contributions
+  de: Letzte Beiträge
+moreButton:
+  en: Show {count} more
+  de: Zeig {count} weitere an / {count} weitere anzeige[n] ?
```
Comment (ahelmberger) [inline src/components/pages/user-profile.yml:9]: {count} weitere anzeigen

Comment (ahelmberger) [reply src/components/pages/user-profile.yml:9]: But, if `count` is 1 then "{count} weitere**n** anzeigen" (meaning: "1 weiteren Beitrag")

**Comment by ahelmberger** [comment]
As the page name has changed we should also flag the ticket and add the necessary config/code updates

File: src/components/pages/user-profile.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/pages/user-profile.yml]: Thanks, changed it so.

**Review by ahelmberger — APPROVED** [review]

**Comment by adrianaluchian** [comment]
> As the page name has changed we should also mark the ticket and add the necessary config/code updates

Not the case (yet).

**Pull request merged**